deploy/prod/common-values-ampc-hnsw.yaml

```diff
@@ -89,76 +89,3 @@ mountSSLCerts:
   enabled: true
   mountPath: /etc/ssl/private
 
-nginxSidecar:
```

Comment:
Prod hnsw is not deployed yet - but we surely won't need nginx, as the decision to go with app-level TLS is set.